package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "js-libs",
   "description": "A collection of JavaScript libraries for dapp development.",
+  "version": "v3.0.26",
```

Comment:
Remove this.

I just tried a `lerna version patch --push false` locally, it bumps all packages correctly, but doesn't bump this.